client/meter-index.html

```diff
@@ -20,7 +25,12 @@
         </div>
 
         <div class="speaker-container beige-border">
-            <div id="speaker"></div>
+          <div id="speaker">
+            <div onclick="switchRange(this)" class="range-indicator long"><p>Long Range</p><div class="knob-light"></div></div>
```

Comment:
Would it be better to use a click handler for these? I imagine you used onclick for convenience. Something else to know is that since we are targeting mobile, we will need touch events not click events.Let  me know if you need any help with that.